prospero-common/src/main/resources/prospero-known-combinations.yaml

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
-1, prospero is a community project and is not the place to provide product specific configuration